pkg/cloudscale/fixtures.go

```diff
@@ -0,0 +1,92 @@
+package cloudscale
+
+import (
+	"database/sql"
+
+	"github.com/appuio/appuio-cloud-reporting/pkg/db"
+)
+
+const (
+	// source format: 'query:zone:tenant:namespace' or 'query:zone:tenant:namespace:class'
+	// We do not have real (prometheus) queries here, just random hardcoded strings.
+	sourceQueryStorage    = "object-storage-storage"
+	sourceQueryTrafficOut = "object-storage-traffic-out"
+	sourceQueryRequests   = "object-storage-requests"
+)
+
+var (
+	// SourceZone represents the zone of the bucket, not of the cluster where the request for the bucket originated.
+	// All the zones we use here must be known to the appuio-odoo-adapter as well.
+	sourceZones = []string{"cloudscale"}
```

Comment:
This constant/variable for exoscale is called provider. I would keep the same naming either `provider` or `sourceZones`. Is there a reason it is an array?

Reply:
not sure - I copied this code verbatim. This part I'm going to refactor and unify as a follow-up if that's fine.

Reply:
Then I'm ok with it.